### PR TITLE
Use different query for Sybase when a boolean column is queried

### PR DESCRIPTION
**Pull request by MarianMacik**
Hi, @sutaakar,

here is the fix for Sybase DB in the query expression. I have tried it with our automation and it works now on all DBs correctly.

Marian

### CODE REVIEW

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java [hunk 1]
```diff
@@ -758,10 +758,18 @@ private QueryDefinition createQueryDefinition(String target) {
     }
 
     private QueryDefinition createErrorsQueryDefinition() {
+        String queryExpression = "select * from ExecutionErrorInfo where ERROR_ACK = ";
+        // Sybase needs special treatment when it comes to boolean
+        if (TestConfig.getKieServerDataSourceDriverClass().startsWith("com.sybase")) {
```
Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java]: Just an idea, this condition could be extracted to separate method to make it more readable. Something like TestConfig.isSybaseDataSource(). That can be reused in the future for other tests.

Comment (rsynek) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java]: +1 and ternary operator can change it into a one-liner

Comment (MarianMacik) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java]: Fixed.

**Comment by MarianMacik** [comment]
Jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Pull request merged**